BatchAI. Update the quickstart to match the latest CLI version

AlexanderYukhanov opened the pull request:
New CLI version allows users to specify --storage-account-name
argument during cluster creation. So, there is no need to deal
with environment variables any more.

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Commit:
BatchAI. Update the quickstart to match the latest CLI version

New CLI version allows users to specify --storage-account-name
argument during cluster creation. So, there is no need to deal
with environment variables any more.

Commit:
Merge branch 'master' into master

AlexanderYukhanov reviewed (commented):
Seems ok to me. How can I as an author review it?

GitHubber17 committed:
Update quickstart-cli.md

GitHubber17 committed:
Update quickstart-cli.md

GitHubber17 committed:
Update quickstart-cli.md
